odoo/tools/pdf.py

```diff
@@ -2,6 +2,41 @@
 # Part of Odoo. See LICENSE file for full copyright and licensing details.
 import io
 from PyPDF2 import PdfFileWriter, PdfFileReader
+from PyPDF2.generic import DictionaryObject, NameObject, ArrayObject, DecodedStreamObject, NumberObject, createStringObject
+from sys import platform
+from subprocess import run
+from logging import getLogger
+from hashlib import md5
+from zlib import compress
+from datetime import datetime
+
+from odoo.tools.misc import file_open
+from odoo.exceptions import UserError
+from odoo.tools.translate import _
+
+_logger = getLogger(__name__)
+DEFAULT_PDF_DATETIME_FORMAT = "D:%Y%m%d%H%M%S+00'00'"
+
+
+def _get_ghostscript_bin():
+    commands = {
+        'win32': ['gswin64c', 'gswin32c'],
+        'darwin': ['gsos2'],
```

Comment:
When installed through homebrew, the binary is `gs`
```suggestion
        'darwin': ['gsos2', 'gs'],
```